src/features/hiro-messages/hiro-messages.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Box, Flex, FlexProps } from '@stacks/ui';
+
+import { useHiroMessages } from '@common/hooks/use-hiro-messages';
+import { HiroMessageItem } from './components/hiro-message-item';
+
+export const HiroMessages = (props: FlexProps) => {
+  const warnings = useHiroMessages();
```

Comment:
`const messages` etc.?

Reply:
thanks for noticing, please check updates